samples/apps/copilot-chat-app/webapp/README.md

```diff
@@ -5,6 +5,59 @@ production deployments.
 
 See [../README.md](../README.md) for complete instructions on setting up and running the application.
 
+## How to use HTTPS for local development
+If you need to run Copilot Chat with HTTPS, you need to create a certificate and sign it with a Certificate Authority (CA) that is trusted locally by your device and browser. You can do that easily with [mkcert](https://github.com/FiloSottile/mkcert#installation).
+
+1. From an elevated shell, run
+    ```
+    choco install mkcert
+    ```
+1. Create a new local CA by running
+    ```
+    mkcert -install
+    ```
+1. Create a new certificate for all hostnames you wish to run the app on. 
+    > It is recommend you do this at the /WebApp directory level. 
+
+    Run
+    ```
+    mkcert localhost 127.0.0.1 example.test
+    ```
+    You should see an output that looks like this:
+    ```
+    Created a new certificate valid for the following names 📜
+        - "localhost"
+        - "127.0.0.1"
+        - "example.test"
+
+    The certificate is at "./localhost+1.pem" and the key at "./localhost+1-key.pem" ✅
+
+    It will expire on 1 September 2025
+    ```
+1. In the [WebApp .env](./.env) file, uncomment lines 11-13 and populate lines 12 and 13 with the certificate and key files generated in the step above.
```

Comment:
Feel free to ignore this comment, but putting specific line numbers "condemns" you to update them later if the file changes...